### ISSUE TITLE
Stuck in 'Updating references'

### ISSUE THREAD

**Issue by matthiasrichter** [alisw/alibuild#432]
After aborting a build process, restarted build is stuck at the package where the build was aborted before.  Debug messages contain the following information.
```bash
DEBUG:ROOT:0: Cleaning up:
DEBUG:ROOT:0: /src/src/alisw/sw/BUILD/61d869c78b08527b452d902d706e8d83603e2d57
DEBUG:ROOT:0: /src/src/alisw/sw/INSTALLROOT/61d869c78b08527b452d902d706e8d83603e2d57
DEBUG:ROOT:0: Updating from tarballs
DEBUG:ROOT:0: Checking for packages already built.
DEBUG:ROOT:0: Glob pattern used: /src/src/alisw/sw/TARS/ubuntu1604_x86-64/SWIG/SWIG-3.0.7-*.ubuntu1604_x86-64.tar.gz
DEBUG:ROOT:0: Packages already built using this version
DEBUG:ROOT:0: 
DEBUG:ROOT:0: Looking for cached tarball in /src/src/alisw/sw/TARS/ubuntu1604_x86-64/store/12/121a446fc14bade4f60f0ddfd4557a861e45c6c6
DEBUG:ROOT:0: No cache tarballs found
DEBUG:ROOT:0: Updating references.
```

**Comment by matthiasrichter** [comment]
follow up: `Updating references` is just the last message and not related. In fact it seems to be stuck in `git clone`, the directory `sw/MIRROR/swig` is existing and seems to contain some inconsistent state which freezes git clone.

So it's difficult to reproduce, unfortunately I have just deleted the directory. I came into this state after rebooting the machine because my login screen was frozen. One thing one could do is add a debug message before the cloning, just for better guidance.

Leaving the issue open as a reminder, but feel free to close it.
